build/generate_template.py

```diff
@@ -15,7 +15,7 @@ def generate_template(template_name, template_params, dest_file, in_zip_file=Fal
     try:
         template_params['encoding_tag'] = '# -*- coding: utf-8 -*-'
         module_name = template_params['metadata'].config['module_name']
-        lookup = TemplateLookup(directories=['build/templates', 'src/{0}/templates'.format(module_name)])
+        lookup = TemplateLookup(directories=['src/{0}/templates'.format(module_name), 'build/templates'])
```

Comment:
This seems to be that `templates` would appear twice in the string. Intentional?

Reply:
I think you misread the code. It's building up a list of two elements wherein the first one will look like 'src/nifake/templates' and the second one will be 'build/templates'.

Reply:
You're right. It appeared to me that `build/templates` was an argument to the format method. Sigh.